ui/app/AppLayouts/Onboarding2/UnblockWithPukFlow.qml

```diff
@@ -47,7 +47,7 @@ SQUtils.QObject {
 
         function finishWithFactoryReset() {
             root.keycardFactoryResetRequested()
-            root.finished()
+            root.finished(false)
```

Comment:
We don't want to proceed with the original flow when we request a Factory reset

Reply:
Would it be better to emit a different signal then?